internals/daemon/api_notices.go

```diff
@@ -253,12 +237,10 @@ func v1PostNotices(c *Command, r *http.Request, _ *UserState) Response {
 	return SyncResponse(addedNotice{ID: noticeId})
 }
 
-func v1GetNotice(c *Command, r *http.Request, _ *UserState) Response {
-	requestUID, err := uidFromRequest(r)
-	if err != nil {
+func v1GetNotice(c *Command, r *http.Request, user *UserState) Response {
+	if user == nil || user.UID == nil {
```

Comment:
I understand that this isn't a requirement of this change, but can you please add a TODO somewhere that mentions notices needs work when we add other authentication mechanisms?

Reply:
Good call -- added a TODO comment in https://github.com/canonical/pebble/pull/434/commits/bc5d32d6acd44977d39c4f92b5290866707103e4